sds/src/secondary_validation/jwt_claims_validator.rs

```diff
@@ -18,34 +18,57 @@ pub enum ClaimRequirement {
 
 #[derive(Serialize, Deserialize, Default, Clone, Debug, PartialEq)]
 pub struct JwtClaimsValidatorConfig {
+    #[serde(default)]
+    pub required_headers: std::collections::BTreeMap<String, ClaimRequirement>,
     #[serde(default)]
     pub required_claims: std::collections::BTreeMap<String, ClaimRequirement>,
```

Comment:
As it's not yet used and this PR adds a new field, I suggest to make the other attribute even more explicit by renaming it `required_payload_claims`:

```suggestion
    pub required_payload_claims: std::collections::BTreeMap<String, ClaimRequirement>,
```

Reply:
I like the parity between `required_headers` and `required_claims`. I thought it kept things short and unambiguous (at least when you see both next to one another). Also, payload is a very generic term so I'm not sure it'd make explicit that there is a `required_headers` if it's not present...